python/requirements.txt

```diff
@@ -1,7 +1,7 @@
 certifi==2017.7.27.1
 chardet==3.0.4
 idna==2.5
-iopipe==0.6.1
+iopipe==1.7.2
```

Comment:
Python agent is on v1.8.2?

Reply:
I'm suggesting we just unpin it, because this repo is massively out of date and we're not gonna want to update it for every release.